.github/labeler.yml

```diff
@@ -1,106 +1,155 @@
 # list of labels and minimatch globs to match to apply the label.
 
 CI:
-- any: ['.cirrus.yml']
-- any: ['.github/dependabot.yml']
-- any: ['.github/labeler.yml']
-- any: ['.github/workflows/*']
-- any: ['.appveyor.yml']
-- any: ['.codecov.yml']
+  - all:
+      - changed-files:
+          - any-glob-to-all-files:
```

Comment:
I think `any-glob-to-any-file` should be used instead of `any-glob-to-all-files`.
In my understanding, if any files that don't match this glob are changed, this label won't be applied when `any-glob-to-all-files` is used.